lib/wrap.js

```diff
@@ -1,22 +1,19 @@
 const { dirname, extname } = require('path');
 const childProcess = require('child_process');
 const { sync: resolve } = require('resolve');
-const getPackageType = require('get-package-type');
 
 const { getConfig } = require('./cfg');
 const hook = require('./hook');
 const { relay, send } = require('./ipc');
 const resolveMain = require('./resolve-main');
 
-// Remove wrap.js from the argv array
-process.argv.splice(1, 1);
-
 const script = process.argv[1];
 const { extensions, fork, vm } = getConfig(script);
 
 if (process.env.NODE_DEV_PRELOAD) {
   require(process.env.NODE_DEV_PRELOAD);
 }
+require('./suppress-experimental');
```

Comment:
I don't think we should be hiding warnings by default.

Reply:
Although I can see now that it gets quite annoying.

Reply:
I was also trying to hold off on implementing this but I figure that this way is probably for the best since it was affecting the test output a lot!